db_setup.sql

```diff
@@ -8,17 +8,57 @@ UserId VARCHAR(50) NOT NULL,
 FirstName VARCHAR(50) NOT NULL,
 LastName VARCHAR(50) NOT NULL,
 PRIMARY KEY (`UserId`)
-) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+);
 
 CREATE TABLE Review (
-ReviewId VARCHAR(50) NOT NULL,
-UserId varchar(50) NOT NULL,
+ReviewId varchar(50) NOT NULL,
 Date DATE,
-Rating VARCHAR(50),
+ResponsivenessRating integer,
+SecurityDepositReturnedRating integer,
+WeekdayVolumeRating integer,
+WeekendVolumeRating integer,
+GreenStProximityRating integer,
+TransportationProximity integer,
+OverallRating integer,
 Description VARCHAR(500),
+UserId varchar(50) NOT NULL,
```

Comment:
idk how this moved back down here but if you could move userId to second again lol